PD-1487 / 25.04 / replace minio plugin and migration content with EoL statement (master) (#3083)
truenas/documentation: content/Solutions/MinioCOREtoSCALE.md

- apps
---

{{< hint type="warning" title="Minio Plugin End of Life" >}}
Due to the interaction between the TrueNAS, Minio, and FreeBSD software lifecycles, the Minio plugin does not generally deploy in TrueNAS 13.0 nor is full data migration of MinIO data from end of life (EoL) deployments of MinIO possible.

> Users are encourage to pursue a manual backup and restore strategy to move data into the fully maintained and updatable MinIO applications available with TrueNAS 24.04 and later.

Reviewer: ```suggestion
Users are encouraged to pursue a manual backup and restore strategy. 
Sidegrade to TrueNAS 24.04 or newer to move data into the available, fully maintained, and updatable MinIO [applications](https://www.truenas.com/docs/truenasapps/).
```

> Enterprise customers are encouraged to contact iX Support to discuss options for guided data migration from the EoL S3 service or Minio Plugin in TrueNAS 13.0 and into supported MinIO Enterprise applications available in TrueNAS 24.04, 24.10, and newer releases.

Reviewer: ```suggestion
Enterprise customers are encouraged to contact iX Support to discuss options for guided data migration from the EoL S3 service or Minio Plugin in TrueNAS 13.0 to the supported [MinIO Enterprise](https://www.truenas.com/docs/truenasapps/enterpriseapps/minio/) applications available in TrueNAS 24.04 and newer releases.
```
